src/main/java/net/openhft/chronicle/core/pool/ClassAliasPool.java

```diff
@@ -190,6 +192,22 @@ private Class<?> doLookup(String name) {
         }
     }
 
+    private boolean banned(String name) {
+        if (name.isEmpty())
```

Comment:
why would name be empty?

Reply:
It shouldn't, that's why it's banned

Reply:
should we throw an exception or assert instead?

Reply:
I could leave this code out, but you would get an IndexOutOfBoundException which could be confusing.

Reply:
All banned class names result in a specific ClassNotFoundRuntimeException